src/opm/io/eclipse/EInit.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+   Copyright 2020 Equinor ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#include <opm/io/eclipse/EInit.hpp>
+#include <opm/common/ErrorMacros.hpp>
+
+#include <algorithm>
+
+namespace Opm { namespace EclIO {
+
+
+EInit::EInit(const std::string &filename) : EclFile(filename)
+{
+    std::string lgrname = "";
```

Comment:
strings are blank on default initialization.

Reply:
Thanks was not sure about this. Changed to default initialization.